src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -29,6 +29,16 @@ public class DurableTaskOptions
         /// </summary>
         public StorageProviderOptions StorageProvider { get; set; }
 
+        /// <summary>
+        /// The section of configuration related to tracing.
+        /// </summary>
+        public TraceOptions Tracing { get; set; } = new TraceOptions();
+
+        /// <summary>
+        /// The section of configuration related to notifications.
+        /// </summary>
+        public NotificationOptions Notifications { get; set; }
```

Comment:
If we default this to `new NotificationsOptions()`, I think we could get rid of alot of the null checks that were added.

Reply:
Defaulted it new NotificationsOptions(), removed most of the null checks that were added